pandas/tests/arrays/test_datetimelike.py

```diff
@@ -508,6 +508,12 @@ def test_take_fill_valid(self, datetime_index, tz_naive_fixture):
             # require NaT, not iNaT, as it could be confused with an integer
             arr.take([-1, 1], allow_fill=True, fill_value=value)
 
+        value = np.timedelta64("NaT", "ns")
+        msg = f"'fill_value' should be a {self.dtype}. Got '{str(value)}'."
```

Comment:
I _think_ normally shouldn't need str in f-string. could this be a numpy bug?

Reply:
i forget exactly which case broke without the extra `str`, but plausibly numpy